examples/annotation_import/video.ipynb

```diff
@@ -40,9 +40,9 @@
         "    * Polyline \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # Frame base nested classifications do not support using the feature"s name to extract ontology features. 
</pre><p>Looks like the single quotes here got substituted</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1031/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>ah shoot, I run a global replacement of single quotes to double quotes but also replaced text quotes </p>